site/docs/tasks/using-opa.md

```diff
@@ -5,6 +5,42 @@ When OPA is used for data governance, it is deployed as a stand-alone service.
 
 For more details on OPA policies please refer to [OPA documentation](https://www.openpolicyagent.org/docs/latest/policy-language/) in particulate to [the basics](https://www.openpolicyagent.org/docs/latest/policy-language/#the-basics) section which explains how a policy is evaluated.
 
+## OPA Policies Syntax
+
+OPA policies for Fybrik are written in rego files and have the following syntax: `rule[{"action": <action>, "policy": <policy>}]` where `policy` is a string describing the action and `action` is a rego object with the follwing form:
+
+```
+{
+	"name": <name>,
+	<property>: <value>,
+	<property>: <value>,
+	...
+}
+```
+
+- `name` is the name of the action. For example: "RedactAction"
+- `property` is the name of the action property as defined in the [Taxonomy](custom-taxonomy.md). For example: "columns".
+
+Here is an example Fybrik rule:
+```
+  rule[{"action": {"name":"RedactAction","columns": column_names}, "policy": description}] {
+        description := "Redact written columns tagged as sensitive in datasets tagged with finance = true. The data should not be stored in `neverland` storage account"
+        input.action.actionType == "write"
+        input.resource.metadata.tags.finance
+        input.action.destination != "neverland"
+        column_names := [input.resource.metadata.columns[i].name | input.resource.metadata.columns[i].tags.sensitive]
+    }
+```
+
+The following is an example of a new value that the rule above can have if it is evaluated to true:
```

Comment:
An example of an object that may be returned by the rule above:

Reply:
Fixed. Thanks